packages/react-app/src/hooks/useBridgeDirection.js

```diff
@@ -15,8 +20,22 @@ export const useBridgeDirection = () => {
     ambLiveMonitorPrefix,
     homeGraphName,
     foreignGraphName,
+    foreignAmbAddress,
   } = bridgeConfig;
 
+  useMemo(
+    async version => {
+      await fetchAmbVersion(foreignAmbAddress, ethersProvider)
+        .then(res => {
+          if (providerChainId === foreignChainId) {
```

Comment:
@alexkeating 
Would it make more sense to reduce a network call and have this check before we call `fetchAmbVersion` ?

Currently this creates a network call every time `providerChainId` changes.
To reduce that we could use `getEthersProvider(foreignChainId)` from `lib/providers` instead of `ethersProvider` from `useWeb3Context`.

Reply:
Also is this an incorrect usage of `useMemo`? Should it rather be `useEffect`?

Or we could have useMemo return the value. Something like:
```
const foreignAmbVersion = useMemo(async () => {});
```

Reply:
Another thought I have is that will this network call be fired every time `useBridgeDirection` is used in all different places?